addOns/wappalyzer/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Fixed
+- A typos in the help content.
```

Comment:
`typo`

Reply:
A typo in the word typo in a PR addressing typos 😆

Reply:
Hahahaha yeah 🤦‍♂️